packages/zkdb/src/sdk/global/zkdatabase-client.ts

```diff
@@ -61,7 +60,8 @@ export class ZKDatabaseClient {
     if (!db) {
       throw new Error('Database name is required');
     }
-    const apiClient = ApiClient.newInstance(apiURL);
+
+    const apiClient = ApiClient.newInstance(apiURL, storage);
```

Comment:
API Client use different instance of `storage` for browser

Reply:
I solved that problem by using optional for storage to get the public api first and then initial another apiClient that have storage that match specific environment

Reply:
Now we using utils to get public environment